apps/ledger-live-desktop/src/renderer/components/PostOnboardingHub/index.tsx

```diff
@@ -1,19 +1,34 @@
-import React, { useCallback, useMemo } from "react";
+import React, { useCallback, useMemo, useEffect } from "react";
 import { Flex } from "@ledgerhq/react-ui";
-import { useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
 import { usePostOnboardingHubState } from "@ledgerhq/live-common/postOnboarding/hooks/index";
 import { clearPostOnboardingLastActionCompleted } from "@ledgerhq/live-common/postOnboarding/actions";
+import { hubStateSelector } from "@ledgerhq/live-common/postOnboarding/reducer";
+import { setKey } from "~/renderer/storage";
 
 import PostOnboardingActionRow from "./PostOnboardingActionRow";
 import { withV3StyleProvider } from "~/renderer/styles/StyleProviderV3";
 
 const PostOnboardingHub = () => {
   const dispatch = useDispatch();
   const { lastActionCompleted, actionsState } = usePostOnboardingHubState();
+  const postOnboardingState = useSelector(hubStateSelector);
   const clearLastActionCompleted = useCallback(() => {
     dispatch(clearPostOnboardingLastActionCompleted());
   }, [dispatch]);
 
+  useEffect(() => {
```

Comment:
2 things: 
- I'm not sure that using `setKey` is the clean/proper way of doing this on LLD, I didn't work on this kind of stuff earlier so let's look into it together tomorrow 👍 
We might have to implement something in `apps/ledger-live-desktop/src/renderer/middlewares/db.js` and also maybe prefix the actions strings with `"DB:"` ...

- Even if `setKey` was the clean/proper way to do this, we should pull this useEffect call way up in the component tree so that it's not only called when the hub is mounted. (The post onboarding state could change while this component isn't mounted and then it means that we are not saving that new state at all).